climpred/metrics.py

```diff
@@ -151,37 +162,58 @@ def __init__(
         self.minimum = minimum
         self.maximum = maximum
         self.perfect = perfect
-        self.proper = proper
 
     def __repr__(self):
         """Show metadata of metric class."""
         return _display_metric_metadata(self)
 
 
+#####################
+# CORRELATION METRICS
+#####################
 def _pearson_r(forecast, reference, dim=None, **metric_kwargs):
-    """
-    Pearson's Anomaly Correlation Coefficient (ACC).
+    """Pearson product-moment correlation coefficient.
+
+    A measure of the linear association between the forecast and observations that
+    is independent of the mean and variance of the individual distributions. This is
+    also known as the Anomaly Correlation Coefficient (ACC) when correlating anomalies.
 
     .. math::
-        ACC = \\frac{cov(f, o)}{\\sigma_{f}\\cdot\\sigma_{o}}
+        corr = \\frac{cov(f, o)}{\\sigma_{f}\\cdot\\sigma_{o}},
+
+    where :math:`\\sigma_{f}` and :math:`\\sigma_{o}` represent the standard deviation
+    of the forecast and observations over the experimental period, respectively.
 
     .. note::
-        Use metric ``pearson_r_p_value`` to get the corresponding pvalue.
+        Use metric ``pearson_r_p_value`` or ``pearson_r_eff_p_value`` to get the
+        corresponding p value.
 
     Args:
-        forecast (xarray object): forecast
-        reference (xarray object): reference
-        dim (str): dimension(s) to perform metric over.
-                   Automatically set by compute_.
-        metric_kwargs: (optional) weights, skipna, see xskillscore.pearson_r
-
-    Range:
-        * perfect: 1
-        * min: -1
+        forecast (xarray object): Forecast.
+        reference (xarray object): Reference (e.g. observations, control run).
+        dim (str): Dimension(s) to perform metric over. Automatically set by compute
+                   function.
+        weights (xarray object, optional): Weights to apply over dimension. Defaults to
+                                           ``None``.
+        skipna (bool, optional): If True, skip NaNs over dimension being applied to.
+                                 Defaults to ``False``.
+
+    Details:
```

Comment:
cannot we somehow make this dynamic from the Metric class? that would be awesome, but can be future PR

Reply:
I'm not sure how to do this in a docstring, but would be really nice.